Automattic/woocommerce-payments
# Fix tracks identity cookie cache

Fixes p1722880351733569-slack-C02CCT3F1QA

#### Changes proposed in this Pull Request








This PR uses the same approach introduced by #8185 to prevent breaking the cache for the products, cart and checkout pages.

#### Testing instructions





* Access any product page while logged out.
* Access the DevTools Application tab.
* Open the Cookies section.
* Delete the `tk_ai` cookie.
* Reload the page.
* Access the DevTools Network tab.
* Look for the page request (usually the first one).
* There should not be any `Set-Cookie` header (this breaks the cache).
* Look for the `/wp-admin/admin-ajax.php` request.
* There should be one with the `get_identity` action and the `wcpay_product_page_view` event.
* Open the Cookies section.
* There should be another `tk_ai` cookie added by the `get_identity` request.
* Do the same steps accessing the cart and checkout pages, the tracks event should match the page you access.

-------------------


**Post merge**
